config/config.exs

```diff
@@ -471,9 +471,18 @@ config :screens, :screens_by_alert,
   screens_last_updated_ttl_seconds: 3600,
   screens_ttl_seconds: 40
 
-config :screens, Screens.ScreenApiResponseCache,
+config :screens, Screens.V3Api.Cache.Realtime,
+  allocated_memory: 50 * 1024 * 1024,
+  gc_cleanup_min_timeout: :timer.seconds(5),
+  gc_interval: :timer.seconds(30),
+  stats: true,
+  telemetry_prefix: ~w[screens v3_api cache]a
+
+config :screens, Screens.V3Api.Cache.Static,
+  allocated_memory: 150 * 1024 * 1024,
```

Comment:
Could be helpful to add comments with reasoning to these config options so people can understand later (and now). Particularly the `allocated_memory` being three times as high for static vs. realtime caching is unclear.

Reply:
I can try to write something useful here, but it was initially pretty vibes-based and then checked by looking at the real memory usage of the ETS tables in development. The static cache ended up at 130-140MB and the realtime at 15-30MB. The main reason I enabled the built-in stats telemetry was so we'd have some way of knowing whether we're evicting too eagerly in production and thus whether we should tweak these numbers. I wish there was a telemetry event emitted when a new generation is created since that would be much more obvious, but it seems like adapter-specific details like that don't really have telemetry.

Reply:
Gotcha, that reasoning makes sense and I see how that's hard to fully put in a comment. Could maybe just add a comment that these values are based on initial testing and should be adjusted as we learn more in the future.